docs/developer-guide/extensions/ui-extensions.md

```diff
@@ -129,32 +129,95 @@ It is also possible to add an optional flyout widget to your extension. It can b
 
 Below is an example of an extension using the flyout widget:
 
-```typescript
+
```

Comment:
```suggestion
```

unnecessary extra new line added (this would also fail `markdownlint`, which I guess isn't running?)